app/javascript/packs/application.js

```diff
@@ -7,7 +7,6 @@
 //= require jquery_ujs
 //= require popper
 //= require bootstrap-sprockets
-//= require bootstrap
 
```

Comment:
_Because I can_
I added it a long time ago.. we didn't need it to say the least. There are also some other stuff I'll remove because I added it just because.

Reply:
Gotcha